docs/website/mmp-errors.md

```diff
@@ -252,6 +259,15 @@ See https://msdn.microsoft.com/en-us/library/x0w2664k.aspx for more information
 
 ### <a name="MM4134">MM4134: Your application is using the '{0}' framework, which isn't included in the MacOS SDK you're using to build your app (this framework was introduced in OSX {2}, while you're building with the MacOS {1} SDK.) This configuration is not supported with the static registrar (pass --registrar:dynamic as an additional mmp argument in your project's Mac Build option to select). Alternatively select a newer SDK in your app's Mac Build options.
 
+### <a name="MT4173"/>MT4173: The registrar can't compute the block signature for the delegate of type {delegate-type} in the method {method} because *.
```

Comment:
s/MT/MM

Reply:
We should figure out a way to not having to write duplicate documentation here 😒

Reply:
@VincentDondain I've now fixed the docs according to your comments.